index.html

```diff
@@ -8,35 +9,35 @@
     <link rel="stylesheet" href="https://harshita-crypto-ui.netlify.app/style.css">
     <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/7fa15c8e24.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
 </head>
+
 <body>
 
     <nav class="navigation">
         <div class="nav-left">
-
             <a href="index.html" class="txt-l fw-black p-s m-s">Crypto&nbsp;Store</a>
-
         </div>
-        <div class="searchbar-input">
 
+        <div class="searchbar-input">
             <span class="icon-input">
                 <i class="fas fa-search icons"></i> Search items here
             </span>
 
             <input type="search" name="searchbar" class="p-s m-s input-box br-s" placeholder="">
-
         </div>
-        <div class="nav-right">
 
+        <div class="nav-right">
             <button class="btn btn-link m-m">
```

Comment:
use margin-med instead

Reply:
will keep in mind during the next project